src/custom/constants/tokens/index.ts

```diff
@@ -28,3 +30,52 @@ export const ADDRESS_IMAGE_OVERRIDE = {
     'https://raw.githubusercontent.com/1Hive/default-token-list/master/src/assets/xdai/0xe91d153e0b41518a2ce8dd3d7944fa863463a97d/logo.png',
   [WETH_XDAI.address]: getTrustImage(WETH_ADDRESS_MAINNET),
 }
+
+export const V_COW: Record<number, Token> = {
+  // TODO: enable once contract addresses are added
+  // [SupportedChainId.MAINNET]: new Token(
+  //   SupportedChainId.MAINNET,
+  //   V_COW_CONTRACT_ADDRESS[1] || '',
+  //   18,
+  //   'vCOW',
+  //   'Virtual CowSwap Token'
+  // ),
+  // [SupportedChainId.XDAI]: new Token(
+  //   SupportedChainId.XDAI,
+  //   V_COW_CONTRACT_ADDRESS[100] || '',
+  //   18,
+  //   'vCOW',
+  //   'Virtual CowSwap Token'
+  // ),
+  [SupportedChainId.RINKEBY]: new Token(
+    SupportedChainId.RINKEBY,
+    V_COW_CONTRACT_ADDRESS[4] || '',
```

Comment:
can/should use the enum as keys here good sir

Reply:
Ahhhh you mean instead of the magic numbers? Right, will do!